npm/cmd/start.go

```diff
@@ -99,10 +98,7 @@ func start(config npmconfig.Config) error {
 
 	// Setting reSyncPeriod
 	minResyncPeriod := time.Duration(config.ResyncPeriodInMinutes) * time.Minute
-
-	// Adding some randomness so all NPM pods will not request for info at once.
-	factor := rand.Float64() + 1
-	resyncPeriod := time.Duration(float64(minResyncPeriod.Nanoseconds()) * factor)
+	resyncPeriod := time.Duration(float64(minResyncPeriod.Nanoseconds()))
```

Comment:
Agreed, random factor can be removed.
https://github.com/kubernetes/kubernetes/pull/71893#discussion_r240956177 

Resync will not relist.